plugins/woocommerce/src/Blocks/BlockTypes/ProductButton.php

```diff
@@ -93,165 +103,167 @@ protected function render( $attributes, $content, $block ) {
 			)
 		);
 
-		if ( $product ) {
```

Comment:
FYI As we check that `$product instanceof WC_Product` above, I could remove this if. So most of the diff below is caused by indentation. You can enable "Hide whitespace" in the GitHub UI to make it easier to review this part.